package.json

```
@@ -67,22 +67,21 @@
     "pre-commit": "lint-staged"
   },
   "scripts": {
-    "unit": "npm-run-all -s unit:*",
-    "unit:esm": "c8 mocha --color --reporter progress --timeout 30000 'tests/lib/**/*.js'",
-    "unit:cjs": "mocha --color --reporter progress --timeout 30000 tests/lib/commonjs.cjs",
-    "test": "npm-run-all -p unit lint",
-    "lint": "eslint .  --report-unused-disable-directives",
-    "fixlint": "npm run lint -- --fix",
     "build": "rollup -c rollup.config.js",
     "build:debug": "npm run build -- -m",
-    "update-version": "node tools/update-version.js",
+    "build:docs": "node sync-docs.js",
```

```suggestion
    "build:docs": "node tools/sync-docs.js",
```

While here, we could fix this script.

---

Good catch.